tests/org.jboss.tools.maven.ui.bot.test/META-INF/MANIFEST.MF

```diff
@@ -19,4 +19,5 @@ Require-Bundle: org.eclipse.core.runtime,
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: Red Hat
-Export-Package: org.jboss.tools.maven.ui.bot.test
+Export-Package: org.jboss.tools.maven.ui.bot.test,

```

Comment:
I guess that you do not need to export this package at all.

Reply:
Fixed